dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -846,6 +839,7 @@ internal class RumViewScopeTest {
                 hasSessionId(fakeParentContext.sessionId)
                 hasLiteSessionPlan()
                 hasReplay(fakeHasReplay)
+                hasReplayStats(ViewEvent.ReplayStats(recordsCount = fakeReplayRecordsCount))
```

Comment:
imo for the readability and to avoid repeating creating the same object we can just create `fakeReplayStats` object, wdyt?